src/Tribe/Rewrite.php

```diff
@@ -436,7 +436,7 @@ public function filter_url_to_postid( $url ) {
 	 * {@inheritDoc}
 	 */
 	protected function get_matcher_to_query_var_map() {
-		$matchers = [
+		$map = [
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 09bb7bf4166430dc4d7ba33ac81a7ab0</sub>